app/serializers/wizard_field_choice_serializer.rb

```diff
@@ -10,7 +10,7 @@ def id
   def i18nkey
     field = object.field
     step = field.step
-    "wizard.step.#{step.id}.fields.#{field.id}.choices.#{id}"
+    "wizard.step.#{step.id}.fields.#{field.id}.choices.#{id}".underscore
```

Comment:
Can all the `.id` fields be dasherized? If not, what about using `.underscore` just on the field that contains to limit the scope?

Reply:
They can. The same problem could apply to the ID values as well. This matches the implementation for the field itself. 🙂